container/kvm/wrappedcmds_test.go

```diff
@@ -203,7 +203,7 @@ func (commandWrapperSuite) TestAutostartMachineFails(c *gc.C) {
 	container := NewTestContainer("aname", stub.Run, nil)
 	err := AutostartMachine(container)
 	c.Assert(stub.Calls(), jc.DeepEquals, []string{"virsh autostart aname"})
-	c.Assert(err, gc.ErrorMatches, "Boom")
+	c.Check(err, gc.ErrorMatches, "failed to autostart domain \"aname\": Boom")
```

Comment:
I usually use backquotes. Not sure what I was thinking...well I do, but prolly not worth the story. Fixed.